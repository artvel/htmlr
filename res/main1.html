<!DOCTYPE html>
<html lang="en">
<head></head>
<body>
    {% include "sub/sub2/sub2-2.html" %}
    {% include "sub/sub2/sub2-2.html" %}
    {% include "sub/sub2/sub2-2.html" %}
    {% include "sub/sub2/sub2-2.html" %}
</body>
</html>