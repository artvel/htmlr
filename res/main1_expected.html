<!DOCTYPE html>
<html lang="en">
<head></head>
<body>
    <div class="sub2-2"></div>
    <div class="sub2-2"></div>
    <div class="sub2-2"></div>
    <div class="sub2-2"></div>
</body>
</html>